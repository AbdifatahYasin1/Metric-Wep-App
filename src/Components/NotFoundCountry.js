import React from 'react';
import { Link } from 'react-router-dom';

function NotFoundCountry() {
  return (
    <div className="bg-gray flex flex-col items-center justify-center h-screen">
      <h1 className="text-4xl font-bold text-gray-800">404</h1>
      <h2 className="text-2xl font-bold text-gray-800">Country Not Found</h2>
      <Link to="/" className="text-blue-500 hover:text-blue-700">
        Go Back
      </Link>
    </div>
  );
}

export default NotFoundCountry;
