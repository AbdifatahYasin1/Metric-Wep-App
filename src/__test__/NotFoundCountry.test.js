// test this component
import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/store/index';
import NotFoundCountry from '../Components/NotFoundCountry';

describe('NotFoundCountry', () => {
  it('renders NotFoundCountry component', () => {
    const m = renderer.create(
      <Provider store={store}>
        <BrowserRouter>
          <NotFoundCountry />
        </BrowserRouter>
      </Provider>,
    ).toJSON();
    expect(m).toMatchSnapshot();
  });
});
