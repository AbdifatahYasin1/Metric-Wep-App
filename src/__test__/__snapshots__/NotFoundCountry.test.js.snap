// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DetailsPage component 1`] = `
<DocumentFragment>
  <div
    class="text-center"
  >
    Country not found
  </div>
</DocumentFragment>
`;
